### PR TITLE
Update page in Connect AWS Gateway “_index”

### PR DESCRIPTION
**Pull request by jcollins-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by jcollins-axway** [comment]
Jira: APIGOV-16135

**Comment by jcollins-axway** [comment]
@kflan-io @sbolosan @dfeldick @vivekschauhan - please review content

File: content/en/docs/central/connect-aws-gateway/_index.md [hunk 1]
```diff
@@ -9,90 +9,52 @@ description: Understand why you would want a connected / managed environment for
...
+Connecting AWS API Gateway to AMPLIFY Central will provide you with a global centralized view of your APIs and their related traffic.
+
+Each AWS Gateway can be represented by an AMPLIFY Central environment allowing you to better filter APIs and their traffic. Supplied with the environment, two agents (Discovery and Traceability) interact with AWS API Gateway and AMPLIFY Central to:
+
+* Detect changes to AWS API Gateway stages and deployments using the Discovery Agent. The Discovery Agent pushes the service configuration as an API service for the environment, which can then be published as a catalog item to be used by consumers to subscribe to the service.
+* Filter the AWS Cloudwatch logs that are related to discovered APIs and prepare the transaction events that are sent to AMPLIFY platform.
+
+### Discovery Agent
+
+The Discovery Agent is used to discover  new deployments and stage updates to existing deployments for publishing related APIs in AMPLIFY Central (either as  a catalog item or as an API service). As part of the deployment package, use the provided cloud formation scripts  to set up the following agent-dependent AWS Services.
```
Comment (sbolosan) [inline content/en/docs/central/connect-aws-gateway/_index.md:23]: extra space between discover and new

**Review by cbordier — APPROVED** [review]
I approve the changes.
@lbadenhop please proceed.

**Review by kflan-io — APPROVED** [review]

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**